presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestSphericalGeoFunctions.java

```diff
@@ -144,4 +150,63 @@ private void assertDistance(String wkt, String otherWkt, Double expectedDistance
     {
         assertFunction(format("ST_Distance(to_spherical_geography(ST_GeometryFromText('%s')), to_spherical_geography(ST_GeometryFromText('%s')))", wkt, otherWkt), DOUBLE, expectedDistance);
     }
+
+    @Test
+    public void testArea()
```

Comment:
Could you add some tests for multi-polygons?

Reply:
some states are multi polygon (like alaska)

Reply:
Great! Are there polygons with holes? If not, let's add some.

Reply:
that was a good suggestion, as I actually had a bug with these